tests/unit/nupic/encoders/date_test.py

```diff
@@ -34,17 +34,17 @@
 
 
 
-#########################################################################
 class DateEncoderTest(unittest.TestCase):
-  '''Unit tests for DateEncoder class'''
+  """Unit tests for DateEncoder class"""
 
 
   def setUp(self):
-    ##TODO: comment and code don't match - weekend?!!
-    # 3 bits for season, 1 bit for day of week, 2 for weekend, 5 for time of day
```

Comment:
No I don't know the history. I'd guess they wrote the doc string, found out it had to be odd so changed to 3 instead of 2. Sounds like you are saying the encoder doc string description of weekend is wrong as well?